src/ecl/eclWatchTree.ts

```diff
@@ -421,7 +439,8 @@ export class ECLWUNode extends Item<ECLWatchTree> {
     }
 
     contextValue(): string {
-        return this._wu.isComplete() ? "ECLWUNodeComplete" : "ECLWUNode";
+        const prot = this._wu.Protected ? "Protected" : "Unprotected";
```

Comment:
Change to `ECLWUNodeProtected` and `ECLWUNode` respectivly.

Reply:
Bump

Reply:
The reasons for this change:
1. Consistency
2. Avoid a context clash with other extensions (not very likely, but...)